index.html

```diff
@@ -45,6 +46,152 @@ <h1>I'm Stanley Enekwe, Glad to see you!</h1>
+          <li>javascript</li>
+        </ul>  
+      <button class="btn">See project</button>    
+      </div>
+    </section>
+    <section class="section1template">
+      <img class="img2" src="img/Portfolio4.svg" alt="gym">
+        <div class="Section1Text">
+          <h2 class="multi">Multi-Post Stories</h2>  
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See project</button>
+        </div>
+    </section>
+    <section class="sectiondiv">
+      <div class="body-text">
+      <h2 class="big-text">About Myself</h2>
+      <p class="b-text">
+        Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <p class="let-connect">
+        <a href="#">LET'S CONNECT</a> 
+      </p>
+      <div class="icon2"> 
+        <ul> 
+          <li><a href="#"><i class="icons fab fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-linkedin-in" aria-hidden="true"></i></a></li>
+          <li><a href="#"><i class="icons fab fa-medium-m" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-github" aria-hidden="true"></i></a></li>
+          <li><a href="#"><i class="icons fab fa-angellist" aria-hidden="true"></i></a></li>
+        </ul> 
+        </div>
+        <button class="btn">Get My Resume</button>
+      </div>
+      <div class="last-div">
+      <div class="last-icons">
+        <h2 class="lang">languages</h2>
+        <img class="img4" src="img/up-brace.svg" alt="arrow">
+      </div>
+      <div class="icon-flex">
+        <div class="languages">
+          <img src="img/javascript.svg" alt="javascript">
+          <h3>javascript</h3>
+        </div>
+        <div class="languages">
+            <img src="img/html-logo.svg" alt="HTML-Logo">
+            <h3>HTML</h3>
+        </div>
+        <div class="languages">
+            <img src="img/css-logo.svg" alt="css-logo">
+            <h3>CSS</h3>
+        </div>
+        </div>
+        <div class="skill-frame">
+          <div class="frame-right">
+            <h2 class="frame">frameworks</h2>
+            <img class="img3" src="img/right-brace.svg" alt="arrow">
+          </div> 
+        <hr>
+          <div class="skill-right"> 
+          <h2 class="skill">skills</h2>
+          <img class="img5" src="img/right-brace.svg" alt="arrow">
+          </div> 
+          <hr>
+        </div>
+      </div>
```

Comment:
- [ ] Please ensure that categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.
- [ ] Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a` li`. Please replace the `div` tags in this section with the `ul` and `li` tags. for semantic reasons.
- [ ] Please follow the figma designs given for this section. See the difference between the two:
- Your design:
![image](https://user-images.githubusercontent.com/52443674/158998761-d42c997d-71b8-4f3e-bc76-77cfda14037c.png)
- Figma design:
![image](https://user-images.githubusercontent.com/52443674/158998945-f4c8dd11-8de5-4621-a386-83a88b406f10.png)

- Based on the above, you could:

- [ ] Change the background of the languages to a more grayish style as shown on figma.
- [ ] Reduce the width of the languages section and still center them.
- [ ] Reduce the border radius of each of the languages section to match that of the design
- [ ] You could add some padding around the section to match the Figma design.